src/python/bcc/usdt.py

```diff
@@ -12,447 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
-import struct
-import re
+from .libbcc import lib, _USDT_CB
 
-from . import BPF
-from . import ProcStat, ProcUtils
+class USDT(object):
+    def __init__(self, pid=None, path=None):
+        if pid:
+            self.pid = pid
+            self.context = lib.bcc_usdt_new_frompid(pid)
+        elif path:
+            self.path = path
+            self.context = lib.bcc_usdt_new_frompath(path)
 
-class USDTArgument(object):
-        def __init__(self, size, is_signed, location,
-                     register=None, constant=None, deref_offset=None,
-                     deref_name=None):
-                self.size = size
-                self.is_signed = is_signed
-                self.location = location
-                self.register = register
-                self.constant = constant
-                self.deref_offset = deref_offset
-                self.deref_name = deref_name
+        if self.context == None:
+            raise Exception("failed to create USDT context") 
 
-        def _normalize_register(self):
-                normalized = self.register
-                if normalized is None:
-                        return None
-                if normalized.startswith('%'):
-                        normalized = normalized[1:]
-                if normalized in USDTArgument.translations:
-                        normalized = USDTArgument.translations[normalized]
-                return normalized
+    def enable_probe(self, probe, fn_name):
+        if lib.bcc_usdt_enable_probe(self.context, probe, fn_name) != 0:
+            raise Exception("failed to enable probe '%s'", probe)
```

Comment:
Should be `%` instead of `,` otherwise we throw a tuple:

```
$ sudo python ../examples/tracing/nodejs_http_server.py 25549
..
Exception: ("failed to enable probe '%s'", 'http__server__request')
```